client/components/quizSingle/styles.js

```diff
@@ -175,3 +165,16 @@ export const NextQuestionBtnStyled = styled(PrimaryButton)`
   margin: 40px auto;
   border: ${props => props.theme.colors.darkGreen} 5px solid;
 `;
+
+export const ContentWrapper = styled.div`
+  width: ${breakpoint("maxWidth")};
+  max-width: calc(100% - 70px);
+  margin: 0 auto;
+  color: ${props => props.theme.colors.light};
+`;
+
+export const SubmitBtnStyled = styled(PrimaryButton)`
```

Comment:
@super-ming since this is an exact copy of the `NextQuestionBtnStyled` on line 163, would you just give that button a more universal name and update to using it for all quiz buttons?

Reply:
I just wasn't sure if we wanted to use a different color for the three buttons, if there's a color palette that we want to adhere to, etc. If not, then we can make them the same for now.

Reply:
Replaced with a generic button.